### PR TITLE
Fabo/1059 test exe

### LINKED ISSUE #1059: Create test if build Voyager starts
UI Version: 0.9.2

Description:

It may happen that production builds behave different then development builds. We might miss some configuration. We should not be releasing bad builds. Therefor we should test in the CI if a build Voyager actually starts.

Plan:
Voyager outputs `[START SUCCESS]` to console if it can start the view thread without errors. In the CI we create a check that starts the actual build Linux version and listens for the success message on the console. This check is already implemented with `yarn test:exe`.
This is not a test for all build versions for all OSs. Testing this is currently not possible and we need to defer this until we have a proper multi OS CI setup.

### PR DESCRIPTION
**Pull request by faboweb**
Closes #1059

Description:

❤️ Thank you!

### CODE REVIEW

**Comment by faboweb** [comment]
Tested here: https://circleci.com/gh/cosmos/voyager/6719

File: app/networks [hunk 1]
```diff
@@ -1 +1 @@
-../builds/testnets
\ No newline at end of file
+/Users/fabo/Development/voyager/builds/testnets
```
Comment (fedekunze) [inline app/networks]: Double checking if `/Users/fabo/` is ok or not

Comment (faboweb) [reply app/networks]: XD thx

File: tasks/publish.js [hunk 2]
```diff
@@ -36,8 +35,6 @@ const publishRelease = ({ notes, tag, token }) =>
   })
 
 async function main() {
-  await build.buildAllPlatforms()
```
Comment (NodeGuy) [inline tasks/publish.js]: How is this program going to publish a release on GitHub it's not longer building it?

File: tasks/releasePullRequest.js [hunk 3]
```diff
@@ -22,7 +22,7 @@ function updateChangeLog(changeLog, newVersion, now) {
 const updatePackageJson = (packageJson, version) =>
   Object.assign({}, packageJson, { version })
 
-const pushCommit = async ({ token, tag, head }) => {
```
Comment (NodeGuy) [inline tasks/releasePullRequest.js]: That should have been caught earlier on CircleCI by the linter; weird.

File: tasks/publish.js (hunk 2 above)
Comment (faboweb) [reply tasks/publish.js]: As of: https://github.com/cosmos/voyager/issues/1059#issuecomment-416956483

Comment (NodeGuy) [reply tasks/publish.js]: I see.

**Pull request merged**